munininfluxdb/commands/fetch.py

```diff
@@ -1,31 +1,22 @@
 #!/usr/bin/env python
 from __future__ import print_function
-import pwd
 import json
 import os
 import sys
-import argparse
 from collections import defaultdict
 
 from munininfluxdb.utils import Symbol
 from munininfluxdb.settings import Defaults
 
 import influxdb
+import storable
 
-try:
-    import storable
-except ImportError:
-    from vendor import storable
+NAME = 'fetch'
+DESCRIPTION = """'fetch' command grabs fresh data gathered by a still running Munin installation and send it to InfluxDB.
 
-try:
-    pwd.getpwnam('munin')
-except KeyError:
-    CRON_USER = 'root'
```

Comment:
`cron` has been extracted to a separate sub-command. So all cron related stuff has been removed from this file and moved to `commands/cron.py`